### ISSUE TITLE
Add pt-BR translation

### ISSUE THREAD

**Issue by gmmcal** [joemasilotti/railsdevs.com#250]
Blocked by #240 / #264

Since there is an ongoing progress to add i18n support, I think it is nice to also add Brazilian Portuguese translation to this project.

Brazil has a huge ruby/rails community and, unfortunately, a small amount of developers is fully proficient with english language. In order to improve expansion to them, it is nice to have `pt-BR` translation too.

I'm up for the task, as soon as #264 is merged

**Comment by joemasilotti** [comment]
#264 is merged! Feel free to start translating :)
